src/pages/InputPage.tsx

```diff
@@ -1,77 +1,71 @@
-import React, {FC, useState} from "react";
-import {BackgroundBlob} from "../components/core/BackgroundBlob.tsx";
-import {AnimatePresence, motion} from "framer-motion";
-import {OptionStep} from "../components/steps/option/OptionStep.tsx";
+import React, { FC, ReactNode, useState } from "react";
+import { motion } from "framer-motion";
+import { CardComponent } from "../components/core/CardComponent.tsx";
+import { options } from "../config/options.tsx";
+import Cookies from "universal-cookie";
+import { ProcessImage } from "../services/processImageService.ts";
 
 export const InputPage: FC = () => {
-  const [step, setStep] = useState('start');
-  // const [image, setImage] = useState<string>();
-  // const [loading, setLoading] = useState<boolean>(false);
+  const [loading, setLoading] = useState(false);
+  const cookies = new Cookies();
 
-  // const handlePhotoTaken = async (photoData: string) => {
-  //     setImage(photoData);
-  //     await handleNextStep(photoData);
-  // };
-
-  const reset = () => {
-    // setImage(undefined);
-    setStep('start');
+  const handleClick = async (prompt: string) => {
+    cookies.set("lastPrompt", prompt, { path: "/" })
```

Comment:
```suggestion
    localStorage.setItem("lastPrompt", prompt)
```
[Link to localStorage API info](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)